src/Client/ClientInterface.php

```diff
@@ -56,4 +56,15 @@ public function importList(array $filters = []);
    */
   public function importEntity($id);
 
+  /**
+   * Posts the remote entity with the given primary ID.
+   *
+   * @param int|string $id
+   *   The ID of the entity to get.
+   *
+   * @return object
+   *   The remote entity object.
+   */
+  public function exportEntity($id);
```

Comment:
The client interface should accept an entity and ID as well. This would be the stdclass entity that we constructed in the Export Manager with all the values necessary to pass to the remote.